openhcl/hcl/src/ioctl/x64.rs

```diff
@@ -66,9 +66,9 @@ impl ProcessorRunner<'_, MshvX64> {
     }
 
     /// Returns the last VTL according to the register page.
-    pub fn reg_page_vtl(&self) -> Option<Vtl> {
+    pub fn reg_page_vtl(&self) -> Option<GuestVtl> {
```

Comment:
Who's still using this? Should we make it not pub, or otherwise harder to reach for when it's not the right tool?

Reply:
It's being used in run_vp for the HypervisorBackedX86 backing (if it thinks it's handling an intercept).